_data/openseeds/ols-9/schedule.yaml

```diff
@@ -0,0 +1,555 @@
+# Schedule for the openseeds ols-9
+---
+timeline:
+- date:
+  description: Call for Application opens
+  details: See the [guidelines and templates](https://github.com/open-life-science/application-forms)
+- date:
+  description: Application webinar
+  type:
+  - Talk
+  - Q&A
+  notes:
+  recording:
+  details: Watch recordings from previous webinars on 
+    [**YouTube**](https://www.youtube.com/playlist?list=PL1CvC6Ez54KBsPT0fhPtkHmBaXR4f8Dqt)
+- date:
+  description: Application Clinic Call
+  type:
+  - Q&A
+  notes:
+  details: At this call, OLS team will be available to provide help if you have any
+    question related to your application
+- date:
+  description: Call for applications closed
+- date:
+  description: Successful applicants announced
+weeks:
+  '00':
+    start: ''
+    calls:
+    - date: September 13, 2023
+      duration: 60 min
+      title: Mentor onboarding group 1
+      type: Mentor
+      talks: []
+    - date: September 14, 2023
+      duration: 60 min
+      title: Mentor onboarding group 2
+      type: Mentor
+      talks: []
+    - date: September 15, 2023
+      duration: 60 min
+      title: Mentor onboarding group 3
+      type: Mentor
+      talks: []
+  '01':
+    start: July 11, 2024
+    calls:
+    - duration: 30 min
+      title: 1st mentor-mentee meeting
+      type: Mentor-Mentee
+      talks: []
+  '02':
+    start: July 18, 2024
+    calls:
+    - date: July 19, 2024
+      time: "09:00"
+      duration: 90 min
+      notes: https://bit.ly/ols-9-week-02
+      title: Welcome to Open Seeds & Tooling for Project Design
+      recording: https://youtu.be/LnqkAtJLHZ4?si=1xPHJV8LaB6xVNtm&t=294
```

Comment:
I removed the links to slides and recordings (that were from OLS-8) in the spreadsheet. Can you rerun the script to update this file and that slides and recording are removed?

Reply:
Updated. Thank you!